.github/workflows/tests_lqcpu_python.yml

```diff
@@ -195,6 +195,13 @@ jobs:
           git log -1 --format='%H'
           git status
 
+      - name: Patching tests following deprecation (remove after v0.39)
+        if: inputs.lightning-version == 'stable' && inputs.pennylane-version != 'stable'
```

Comment:
Shouldn't pennylane version be 'latest' as we're trying to fix the issue with stable/latest?

Reply:
I didn't have much time to think about that.
This non-pervasive patch only does something if something needs to be done.
I can change it if this is important for you, though.
Just let me know. 🙂

Reply:
I only wonder about the correctness of this condition. We need to ensure that it won't break other checks in the plugin-matrix. If you're confident about it and have already fully checked the plugin-matrix, I won't have any concern with this as we will remove it in the next release.

Reply:
Do you know why we still get lgpu file-name errors in stable/stable? 
- https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/10998801618
- https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/10998790742

Reply:
I was avoiding triggering the CIs without need. But I have no problem changing.

Reply:
Regarding the lgpu file names I have no idea.

Reply:
@maliasadi, I checked the lgpu name situation. I'm not touching these names here, but apparently, GH thinks these files still have the wrong name in this branch. I'm positive (but not 100% sure) that if we merge it to master no change will propagate. I never saw these before.

Reply:
Thanks for checking @AmintorDusko! How did you test the support 3 days ago though? :thinking: 
https://github.com/PennyLaneAI/pennylane-lightning/actions/workflows/compat-check-stable-latest.yml
(I'd like us to double-check if there are more issues that should be fixed)

Reply:
[08f9487](https://github.com/PennyLaneAI/pennylane-lightning/pull/920/commits/08f94879b39c4a5a5b4752f37f4e35375df520b9) :partying_face: